mne/label.py

```diff
@@ -792,9 +788,86 @@ def center_of_mass(self, subject=None, restrict_vertices=False,
                                  subject, subjects_dir, restrict_vertices)
         return vertex
 
+    @verbose
+    def distances_to_edge(self, subject=None, subjects_dir=None,
+                          surface='white', *, verbose=None):
+        """Compute the distance from each vertex to the edge of the label.
```

Comment:
edge with cortical mesh has an ambiguous meaning. Maybe use boundary or border instead of edge here?